setup.py

```diff
@@ -4,17 +4,20 @@
 
 from chirp import CHIRP_VERSION
 
-desktop_files = glob("share/*.desktop")
-image_files = glob('share/*.png')
-stock_configs = glob("stock_configs/*")
-
 setup(name='chirp',
       descrption='A cross-platform cross-radio programming tool',
       packages=find_packages(exclude=["tests", "tests.*"]),
+      include_package_data=True,
       version=CHIRP_VERSION,
       url='https://chirp.danplanet.com',
-      python_requires=">=3.3,<4",
-      install_required=['wxPython', 'serial', 'six', 'future'],
```

Comment:
This may have been a helpful typo. The reason the test jobs are failing with missing `attrdict3` is a known problem with wxPython on Linux only:

https://github.com/wxWidgets/Phoenix/issues/2225

That's why I have the version pins in `requirements.txt`. My understanding is that you're not supposed to specify versions in `install_requires`, and only use package names. However, issues like this show how that will never work.

But, even without that problem, it will make us try to _compile_ wxwidgets itself during test environment setup on linux, which would be supremely unfortunate.

Reply:
The wxPython issue applies to all Unix-like systems (including FreeBSD that I'm using) that did not apply wxWidgets/Phoenix#2224. [Versions can absolutely be specified in `install_requires`](https://peps.python.org/pep-0508/), but proceed with caution from a practical point of view.

Reply:
Yeah, I know it's possible. I'm referring to this recommendation against it:

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#install-requires-vs-requirements-files

> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

Reply:
> But, even without that problem, it will make us try to _compile_ wxwidgets itself during test environment setup on linux, which would be supremely unfortunate.

The current wxPython versions, ie those not available in the Ubuntu 22.04 LTS Jammy repositories, require wxGTK 3.2, also not available in the repositories. Later versions are only available in Focal. That's the supreme unfortunate part.

Reply:
I'm not sure what you mean...I have this running locally on jammy (and focal) using distro packages for wxpython.